docs/CONTRIBUTING.md

```diff
@@ -61,24 +61,21 @@ done the following things first:
 
 1. You have ensured the pull request is rebased on a recent version of your
    respective branch or the latest upstream has been merged.
-2. All of the following commands completed without errors.  You can pass whatever features are relevant for your change. CI will test with all features enabled.
-   * `rustup update stable nightly`
-   * `cargo +nightly fmt --all`
-   * `cargo +nightly clippy --workspace --all-targets --features empty -Z unstable-options`
-   * `cargo test --all-targets --workspace --features empty`
-   * `cargo run --example $YOUR_EXAMPLE --features $YOUR_BACKEND`
+2. All of the following commands completed without warnings or errors.  You can pass whatever features are relevant for your change. CI will test with all features enabled.
```

Comment:
I added a note about using the git hook to run these commands.  Keeping them up to date in at least three places (CI, here, git hook) is kind of painful.  If I make the git hook script a little nicer it run on demand, but you only really need it at pre-commit or pre-push time.

The extra indentation nests the bullets within the numbered item instead of being at the same level as the numbered item visually.  I use https://github.com/joeyespo/grip to edit markdown locally and check against github formatting.